market_maker_keeper/order_book.py

```diff
@@ -440,7 +440,7 @@ def func():
                         self._order_ids_cancelled.add(order_id)
                         self._order_ids_cancelling.remove(order_id)
             except BaseException as exception:
-                self.logger.exception(exception)
+                self.logger.exception(f"Failed to cancel {order_id} ({exception})")
```

Comment:
logging the exception explicitly in not needed as the logging.exception() already handle that : `self.logger.exception(f"Failed to cancel {order_id}")` should be enough.

https://docs.python.org/3/library/logging.html#logging.exception

Reply:
thanks, changed!

Reply:
Thanks, changed!